roles/entitlement_inspect/tasks/main.yml

```diff
@@ -20,6 +20,10 @@
   command: oc describe MachineConfigPool/worker
   failed_when: false
 
+- name: Get the description of the master MachineConfigPool
+  command: oc describe MachineConfigPool/master
+  failed_when: false
```

Comment:
can you store the description in `{{ artifact_extra_logs_dir }}mcp_master.descr` (and same for `worker` above)?
that makes the logs easier to read

Reply:
could you write this into `        > {{ artifact_extra_logs_dir }}/master_MachineConfigPool.desc` (don't forget to also change the task from `command` to `shell`)